app/test_main.py

```diff
@@ -1,3 +1,39 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, goals",
+    [pytest.param(14, 14, [0, 0],
+                  id="should return first 15 dog and cat years"),
+     pytest.param(28, 29, [3, 3],
+                  id="should return every 5 next dog and cat years"),
```

Comment:
The expected result `[3, 3]` for `cat_age=28` and `dog_age=29` seems incorrect. According to the conversion rules, 28 cat years should convert to 3 human years (15 + 9 + 4), and 29 dog years should convert to 3 human years (15 + 9 + 5). Please verify this calculation.